c2corg_ui/templates/waypoint/edit.html

```diff
@@ -7,139 +9,774 @@
+            </li>
+          </ul>
+        </div>
+
+        ## CLIMBING STYLES
+        <div class="form-group data half" ng-if="waypoint.waypoint_type == 'climbing_outdoor' || waypoint.waypoint_type == 'climbing_indoor'">
+          <label translate>climbing_styles</label>
+          <ul class="types">
+            <li ng-repeat="type in ${climbing_styles}"  ng-click="editCtrl.pushToArray(waypoint, 'climbing_styles', type, $event)">
+              <div class="checkbox"><input type="checkbox" ng-checked="waypoint.climbing_styles.indexOf(type) > - 1"><span translate>{{type}}</span></div>
+            </li>
+          </ul>
+        </div>
+
+        ## CLIMBING INT TYPES
+        <div class="form-group data half" ng-if="waypoint.waypoint_type == 'climbing_indoor'">
+          <label translate>climbing_indoor_types</label>
+          <ul class="types">
+            <li ng-repeat="type in ${climbing_indoor_types}"  ng-click="editCtrl.pushToArray(waypoint, 'climbing_indoor_types', type, $event)">
+              <div class="checkbox"><input type="checkbox"  ng-checked="waypoint.climbing_indoor_types.indexOf(type) > - 1"><span translate>{{type}}</span></div>
+            </li>
+          </ul>
+        </div>
+
+
+        ## ROCK TYPES
+        <div class="form-group data half" ng-if="waypoint.waypoint_type == 'climbing_outdoor'">
+          <label translate>rock_types</label>
+          <ul class="types">
+            <li ng-repeat="type in ${rock_types}"  ng-click="editCtrl.pushToArray(waypoint, 'rock_types', type, $event)">
+              <div class="checkbox"><input type="checkbox"  ng-checked="waypoint.rock_types.indexOf(type) > - 1"><span translate>{{type}}</span></div>
+            </li>
+          </ul>
+        </div>
+
+        ## PRODUCT TYPES
+        <div class="form-group data half" ng-if="waypoint.waypoint_type == 'local_product'">
+          <label><span translate>product_types</span><span ng-if="waypoint.waypoint_type == 'local_product'"> *</span></label>
+          <ul class="types">
+            <li ng-repeat="type in ${product_types}"  ng-click="editCtrl.pushToArray(waypoint, 'product_types', type, $event)">
+              <div class="checkbox"><input type="checkbox"><span translate>{{type}}</span></div>
+            </li>
+          </ul>
+        </div>
       </div>
-    </div>
-    <div id="longitude-group" class="form-group" 
-         ng-class="{ 'has-error': editForm.longitude.$touched && editForm.longitude.$invalid,
-                                'has-success': editForm.longitude.$valid }">
-      <label><span translate>Longitude</span> *</label> 
-     <div class="input-group">
-        <input type="number" name="longitude" ng-model="waypoint.lonlat.longitude" ng-change="editCtrl.updateMap()" class="form-control" required min="-180" max="180"/>
-        <span class="input-group-addon">°E</span>
+
+    </section>
+  </div>
+
+
+      <div class="step two">
```

Comment:
The indentation of the whole following block looks weird. You should probably shift it leftward a bit.